packages/mobile/src/home/reducers.ts

```diff
@@ -17,6 +17,7 @@ export interface Notification {
   minVersion?: string
   maxVersion?: string
   countries?: string[]
+  blacklist?: string[]
```

Comment:
We should avoid `blacklist`, see https://tools.ietf.org/id/draft-knodel-terminology-00.html#rfc.section.1.2
How about `blockedCountries`?

Reply:
Thanks for pointing that out, I have often wondered about the usage of that term. Will make that change!